.evergreen/evergreen.yml.in

```diff
@@ -1266,6 +1262,8 @@ buildvariants:
   - name: e2e_rhel90_arm64
     display_name: "RHEL 9.0 arm64 (E2E Tests)"
     run_on: rhel90-arm64-small
+    expansions:
+      disable_openssl_shared_config: true
```

Comment:
Can we name this in a way that makes it clear that it only affects the static/bundled-OpenSSL build?

Reply:
I tried something in [5a8818b](https://github.com/mongodb-js/mongosh/pull/1542/commits/5a8818be8467cbb38ddfb9ecf52bbb655166b019). Let me know if that looks ok?